src/Modules/Backend/config/services.yaml

```diff
@@ -28,3 +28,6 @@ services:
 
   ForkCMS\Modules\Backend\Console\:
     resource: '../Console/*'
+
+  ForkCMS\Modules\Backend\Domain\Authentication\TwoFactorAuthenticationFactory:
```

Comment:
why does this need to be public? Can't you inject it?

Reply:
No we use this in scheb_2fa.yaml to enable/disable the services.